### PR TITLE
feat(BUX-421): replace router logger

### PR DESCRIPTION
**Pull request by pawellewandowski98**
# Description
Unify logs in http router

# Pull Request Checklist

- [ ] 📖 I created my PR using provided  : [CODE_STANDARDS](https://github.com/BuxOrg/bux-server/blob/main/.github/CODE_STANDARDS.md)
- [ ] 📖 I have read the short Code of Conduct: [CODE_OF_CONDUCT](https://github.com/BuxOrg/bux-server/blob/main/.github/CODE_OF_CONDUCT.md)
- [ ] 🏠 I tested my changes locally.
- [ ] ✅ I have provided tests for my changes.
- [ ] 📝 I have used conventional commits.
- [ ] 📗 I have updated any related documentation.
- [ ] 💾 PR was issued based on the Github or Jira issue.

### CODE REVIEW

File: actions/middleware.go [hunk 1]
```diff
@@ -101,3 +101,33 @@ func CheckAuthentication(appConfig *config.AppConfig, bux bux.ClientInterface, r
...
+		start := time.Now()
+
+		// Fire the request
+		h(writer, req, ps)
+
+		// Complete the timer and final log
+		elapsed := time.Since(start)
+
+		if a.AppConfig.RequestLogging {
+			a.Services.Logger.Debug().Msgf("%d | %s | %s | %s | %s ", writer.Status, elapsed, req.RemoteAddr, req.Method, req.URL)
```
Comment (dorzepowski) [inline actions/middleware.go:130]: Additionally we could add those params in contextual fields and maybe some additional marker like Str("logger", "http-request")

**Review by dorzepowski — CHANGES_REQUESTED** [review]

File: actions/middleware.go (hunk 1 above)
Comment (pawellewandowski98) [reply actions/middleware.go:130]: So I should leave it in a message and additionally add to context?

File: actions/middleware.go [hunk 2]
```diff
@@ -101,3 +101,33 @@ func CheckAuthentication(appConfig *config.AppConfig, bux bux.ClientInterface, r
...
+		// Start the log (timer)
+		start := time.Now()
+
+		// Fire the request
+		h(writer, req, ps)
+
+		// Complete the timer and final log
+		elapsed := time.Since(start)
+
+		if a.AppConfig.RequestLogging {
```
Comment (arkadiuszos4chain) [inline actions/middleware.go:129]: Do we need `RequestLogging` flag if we log it as DEBUG?

Comment (pawellewandowski98) [reply actions/middleware.go:129]: We can remove it If we want, It was still in config after refactoring so I used it

File: actions/middleware.go (hunk 1 above)
Comment (pawellewandowski98) [reply actions/middleware.go]: Done, I added all parameters into logger

File: actions/middleware.go (hunk 2 above)
Comment (arkadiuszos4chain) [reply actions/middleware.go:129]: Leave it as it is

**Review by arkadiuszos4chain — APPROVED** [review]

**Review by Nazarii-4chain — APPROVED** [review]

**Review by dorzepowski — APPROVED** [review]

**Pull request merged**